### PR TITLE
fix css style checkbox (form : edit user)

### PR DESCRIPTION
**Pull request by qmonmert**
- Please make sure the below checklist is followed for Pull Requests.

- [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [ ] Tests are added where necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/client/templates/react/src/main/webapp/app/modules/administration/user-management/user-management-model.tsx [hunk 1]
```diff
@@ -103,7 +103,7 @@ export class UserManagementModel extends React.Component<IUserManagementModelPro
               <AvFeedback>This field is required to be at least 5 characters.</AvFeedback>
             </AvGroup>
             <AvGroup>
-              <Label check inline>
+              <Label check inline className="form-check-label">
```
Comment (deepu105) [inline generators/client/templates/react/src/main/webapp/app/modules/administration/user-management/user-management-model.tsx]: @qmonmert adding this messes up the alignment
image

Comment (deepu105) [reply generators/client/templates/react/src/main/webapp/app/modules/administration/user-management/user-management-model.tsx]: so i'm not sure if its a good idea

Comment (deepu105) [reply generators/client/templates/react/src/main/webapp/app/modules/administration/user-management/user-management-model.tsx]: Its needed only if your checkbox is placed before the label, here it is after the label

Comment (qmonmert) [reply generators/client/templates/react/src/main/webapp/app/modules/administration/user-management/user-management-model.tsx]: With Chrome it renders like this
capture d ecran 2017-12-06 a 16 09 16

**Comment by qmonmert** [comment]
@deepu105 it seems working well with : 

`
<AvGroup>
              <Label check>
                <AvInput type="checkbox" name="activated" /> <Translate contentKey="userManagement.activated">Activated</Translate>
              </Label>
              <AvFeedback>This field is required.</AvFeedback>
              <AvFeedback>This field cannot be longer than 100 characters.</AvFeedback>
              <AvFeedback>This field is required to be at least 5 characters.</AvFeedback>
</AvGroup>
`
Can I make an other PR with this ?

**Pull request closed without merging**